spec/unit/installer/user_project_integrator/target_integrator_spec.rb

```diff
@@ -299,6 +299,23 @@ module Pod
           phase.output_paths.should.be.empty
         end
 
+        it 'clears input and output paths from script phase if it exceeds limit' do
+          # The paths represented here will be 500 for input paths and 500 for output paths which will exceed the limit.
```

Comment:
it's actually 501 for each, which is why it's exceeding the threshold. If it was exactly 500 + 500, we wouldn't be clearing the list

Reply:
yes and to say that I had a comment because of the `.sh` script being part of these paths. Will amend comment.

Reply:
well I mean that its actually 502 because of the `.sh` file

Reply:
updated